setup.cfg

```diff
@@ -75,7 +75,7 @@ base-runtime =
     urllib3>=2.0.7
     Werkzeug>=3.0.0
     xmltodict>=0.13.0
-    rolo>=0.3,<0.4
```

Comment:
> Currently, we can rely on the pins to keep us diligent on updating the lower bound of the version as well.
I don't think this is true, tbh.

> How will we handle increasing lower bounds of versions in the future?
We didn't really do that in the past, and there also isn't a need for this for runtime dependencies in my opinion.

Reply:
> Currently, we can rely on the pins to keep us diligent on updating the lower bound of the version as well.

Actually, in this specific case as an exception, this is true. And until recently it was useful to define upper version boundary (f.e. when trying to build an older version in the future). But now we preserve the actually tested version in the requirement lock files anyways, so I think we should just handle this the same way we handle all other dependencies:
- Avoid any upper version limits if possible.
- Update the pins regularly.
- If we explicitly upgrade to a new version (a new release of `rolo` was released and we want to upgrade to it immediately, instead of waiting for the update PR). We will do so in an explicit PR. If we explicitly upgrade our code to be compatible with a new version, we can increase the minimal version to reflect the technical minimal version.